### PR TITLE
Add basic game functionality, start work gen and tile handling

### PR DESCRIPTION
**Pull request by Tacitor**

### CODE REVIEW

File: KrampitzKreutzwiserSettlersOfCatan/src/dataFiles/OldCode.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
+/*
```
Comment (Tacitor) [inline KrampitzKreutzwiserSettlersOfCatan/src/dataFiles/OldCode.java:1]: Don't worry about this being here, I just didn't want to fully delete it just yet.

File: KrampitzKreutzwiserSettlersOfCatan/src/dataFiles/tilePos.txt [hunk 2]
```diff
@@ -0,0 +1,56 @@
+661
```
Comment (Tacitor) [inline KrampitzKreutzwiserSettlersOfCatan/src/dataFiles/tilePos.txt:1]: The x and y coords for the 19 tiles

**Review by Tacitor — COMMENTED** [review]
### Ready for you to approve Evan. If you have **any** questions about any line fo code please ask.

File: KrampitzKreutzwiserSettlersOfCatan/src/krampitzkreutzwisersettlersofcatan/GameFrame.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        
+        initFrame();
+    }
+    
+    /**
+     * Set up the JFrame
+     */
+    private void initFrame() {
+        setTitle("Settlers of Catan");
+        setSize(1920, 1080); //set the size to 1080p
```
Comment (Evan-Kreutzwiser) [inline KrampitzKreutzwiserSettlersOfCatan/src/krampitzkreutzwisersettlersofcatan/GameFrame.java:28]: There's got to be a better way to do this

**Review by Evan-Kreutzwiser — APPROVED** [review]

**Pull request merged**